lib/block-supports/typography.php

```diff
@@ -228,6 +228,30 @@ function gutenberg_typography_get_css_variable_inline_style( $attributes, $featu
 	return sprintf( '%s:var(--wp--preset--%s--%s);', $css_property, $css_property, $slug );
 }
 
+/**
+ * Renders typography styles/content to the block wrapper.
+ *
+ * @param  string $block_content Rendered block content.
+ * @param  array  $block         Block object.
+ * @return string                Filtered block content.
+ */
+function gutenberg_render_typography_support( $block_content, $block ) {
+	$custom_font_size = isset( $block['attrs']['style']['typography']['fontSize'] ) ? $block['attrs']['style']['typography']['fontSize'] : null;
```

Comment:
Tiny nit (feel free to ignore if this isn't appropriate): should we return early with `return $block_content` if `$custom_font_size` is null / empty, rather than calling `get_typography_font_size_value`? A possible example, instead of using a ternary here, we might front-load the function with an if statement:

```
if ( ! isset( $block['attrs']['style']['typography']['fontSize'] ) ) {
  return $block_content;
}
```

Reply:
I like. Will do. 
Thx!